src/main/java/org/gradle/profiler/GradleBuildInvocationResult.java

```diff
@@ -39,36 +78,28 @@ public Duration extractFrom(GradleBuildInvocationResult result) {
         }
     };
 
-    public static Sample<GradleBuildInvocationResult> sampleBuildOperation(String buildOperationDetailsClass) {
-        return new Sample<GradleBuildInvocationResult>() {
-            @Override
-            public String getName() {
-                return getSimpleBuildOperationName(buildOperationDetailsClass);
-            }
+    public static final Sample<GradleBuildInvocationResult> GRADLE_TOOLING_AGENT_EXECUTION_TIME = new Sample<GradleBuildInvocationResult>() {
+        @Override
+        public String getName() {
+            return "Gradle execution time";
+        }
 
-            @Override
-            public Duration extractFrom(GradleBuildInvocationResult result) {
-                return result.cumulativeBuildOperationTimes.getOrDefault(buildOperationDetailsClass, Duration.ZERO);
-            }
-        };
-    }
+        @Override
+        public Duration extractFrom(GradleBuildInvocationResult result) {
+            return result.gradleToolingAgentExecutionTime;
+        }
+    };
 
-    public GradleBuildInvocationResult(
-        BuildContext buildContext,
-        Duration executionTime,
-        @Nullable Duration garbageCollectionTime,
-        @Nullable Duration timeToTaskExecution,
-        Map<String, Duration> cumulativeBuildOperationTimes,
-        String daemonPid
-    ) {
-        super(buildContext, executionTime);
-        this.garbageCollectionTime = garbageCollectionTime;
-        this.timeToTaskExecution = timeToTaskExecution;
-        this.cumulativeBuildOperationTimes = cumulativeBuildOperationTimes;
-        this.daemonPid = daemonPid;
-    }
+    public static final Sample<GradleBuildInvocationResult> STUDIO_EXECUTION_TIME = new Sample<GradleBuildInvocationResult>() {
```

Comment:
```suggestion
    public static final Sample<GradleBuildInvocationResult> IDE_EXECUTION_TIME = new Sample<GradleBuildInvocationResult>() {
```

Should this also be renamed? As well as the `getName()` method?

Reply:
Renamed.